bucket/aurora.json

```
@@ -0,0 +1,24 @@
+{
+    "homepage": "http://www.project-aurora.com/",
+    "description": "Unified lighting effects across multiple brands and various games",
```

With this generic description is not straightforward, what lighting effects are meant.

```suggestion
    "description": "Unified peripherals lighting effects across multiple brands and various games",
```